extensions/amp-next-page/amp-next-page.md

```diff
@@ -254,8 +330,13 @@ may make a request to something like
 `https://foo.com/config.json?0.8390278471201` where the RANDOM value is
 randomly generated upon each impression.
 
+## Accessibility
+
+The default footer and default separator both present a generic, non-localized `aria-label` describing their content. If this label is not satisfactory, please consider using a custom footer/separator element to improve accessibility.
```

Comment:
Thanks so much for adding this section :)

Update `footer/separator` to `footer or separator`

Reply:
@wassgha this update has not been made.